custom_components/trakt_tv/schema.py

```diff
@@ -49,6 +49,7 @@ def upcoming_schema() -> Dict[str, Any]:
         subschemas[trakt_kind.value.identifier] = {
             Required("days_to_fetch", default=30): cv.positive_int,
             Required("max_medias", default=3): cv.positive_int,
+            Required("show_next_episode", default=False): bool,
```

Comment:
Hummm, you're right,  I'm not good enough in python for HA to do it in a short period of time, but I'll try to see if I can manage to have it done. Otherwise don't hesitate to complete that :) Let me know!

Reply:
Remove it, it is not necessary anymore